### DISCUSSION TITLE
Welcome to postgres-codespaces DISCUSIONES GOOD!

### DISCUSSION THREAD

**Discussion by nicosistemas** [nicosistemas/postgres-codespaces#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tarq008178** [comment]
con este usuario puedo comentar, si está como contributor o collaborator

acá hice una table con / tabla de markdown
| Header |
|--------|
| Cell |
| Cell |
| Cell |
| Cell |
